encoding/protobuf.go

```diff
@@ -54,8 +55,18 @@ func (p protoSerializer) Encoding() Encoding {
 }
 
 func (p protoSerializer) Request(body []byte) (*transport.Request, error) {
+	m, err := unmarshal.YAML(body)
+	if err != nil {
+		return nil, err
+	}
+
+	json, err := json.Marshal(m)
+	if err != nil {
+		return nil, err
```

Comment:
should be able to pass in yaml data that cannot be converted to JSON (e.g., map with integers).